nixos/modules/services/web-servers/nginx/default.nix

```
@@ -446,6 +446,24 @@ in
         ";
       };
 
+      defaultSSLPort = mkOption {
+        type = types.port;
+        default = 443;
+        example = literalExpression ''443'';
```

We only use `''` when you need multiline strings with contextual indentation

```suggestion
        example = literalExpression "443";
```

---

using `literalExpression` isn't necessary here anyway, numeric literals can just be written as is.